spec/transaction/paths.yaml

```diff
@@ -789,6 +942,249 @@ paths:
+        "404":
+          $ref: "../components/responses.yaml#/components/responses/NotFound"
+        "5XX":
+          $ref: "../components/responses.yaml#/components/responses/ServerError"
+    post:
+      tags:
+        - Collection Asset Upload Management
+      summary: Create a new collection asset multipart upload
+      description: |
+        Create collection asset multipart upload.
+
+        A file part must be at least 5 MB except for the last one and at most 5 GB, otherwise the complete operation will fail.
+
+        *Note: in order to provide integrity check during the upload, the base64-encoded 128-bit MD5 digest of each part must be
+        computed and passed in the create endpoint. Then this digest must also be passed as `Content-MD5` header during the upload.*
+      operationId: createCollectionAssetUpload
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: "./components/schemas.yaml#/components/schemas/assetUploadCreate"
+      responses:
+        201:
+          description: Created collection asset multipart upload
+          headers:
+            Location:
+              description: A link to the collection asset multipart upload object
+              schema:
+                type: string
+                format: url
+          content:
+            application/json:
+              schema:
+                $ref: "./components/schemas.yaml#/components/schemas/assetUploadCreate"
+        "400":
+          $ref: "../components/responses.yaml#/components/responses/BadRequest"
+        "404":
+          $ref: "../components/responses.yaml#/components/responses/NotFound"
+        "409":
+          description: Another upload is already in progress, you need first to complete or abort it before creating a new upload.
+          content:
+            application/json:
+              schema:
+                $ref: "./schemas.yaml#/components/schemas/uploadInProgress"
+        "5XX":
+          $ref: "../components/responses.yaml#/components/responses/ServerError"
+  "/collections/{collectionId}/assets/{assetId}/uploads/{uploadId}":
+    parameters:
+      - $ref: "../components/parameters.yaml#/components/parameters/collectionId"
+      - $ref: "./components/parameters.yaml#/components/parameters/assetId"
+      - $ref: "./components/parameters.yaml#/components/parameters/uploadId"
+    get:
+      tags:
+        - Collection Asset Upload Management
+      summary: Get a collection assets multipart upload
+      description: |
+        Return a collection assets multipart upload.
+      operationId: getCollectionAssetUpload
+      parameters:
+        - $ref: "../components/parameters.yaml#/components/parameters/IfMatch"
+        - $ref: "../components/parameters.yaml#/components/parameters/IfNoneMatch"
+      responses:
+        "200":
+          description: Collection asset multipart upload description.
```

Comment:
Looks like a stub description that has not been changed yet.

Reply:
Now it is changed to

> Specified collection asset multipart upload.

I am not sure I understand what this means. Shouldn't it be something like "Did X successfully"? For example, `completeCollectionAssetMultipartUpload` has description "Asset multipart upload completed successfully."

Reply:
Ok, fine with me :+1: